coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -0,0 +1,35 @@
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.results.result_actions.ResultAction import ResultAction
+import os.path
+
+
+class IgnoreResultAction(ResultAction):
+
+    SUCCESS_MESSAGE = "Result ignored successfully."
```

Comment:
Just understood (after reading the code) that it adds the ignore keyword there. This should be mentioned more explicitly.